deploy/deploy.sh

```diff
@@ -50,7 +84,7 @@ migrate_dokuwiki_data() {
     fi
 
     echo "Copying ${WIKIDATA}/data/${dir}..."
-    sudo cp -r ${WIKIDATA}/data/${dir}/* deploy/wiki_config/dokuwiki/data/${dir}/
+    sudo cp -r "${WIKIDATA}/data/${dir}/"* "deploy/wiki_config/dokuwiki/data/${dir}/"
```

Comment:
Good catch.  I got kind of lazy with path quoting.

Reply:
This was thanks to `shellcheck` :)